frontend/src/main-components/InsideFolderContent.js

```diff
@@ -1,23 +1,57 @@
-import React from "react";
-import styled from "@emotion/styled";
-
+import React from 'react';
+import styled from '@emotion/styled';
+import Flashcard from '../flashcards/Flashcard.js';
+import NewCard from '../flashcards/NewCard.js';
+import { LogDebugMessage } from './LogDebugMessage.js';
 
 const InsideFolderContent = (props) => {
-  const Container = styled.div`
-    flex: 9;
-    display: flex;
-    border: 1rem solid white;
-    border-radius: 1rem;
-    background-color: white;
-    margin: 1%;
-  `;
+	const Container = styled.div`
+		flex: 9 1 auto;
+		display: flex;
+		border: 1rem solid white;
+		border-radius: 1rem;
+		background-color: white;
+		margin: 1%;
+		flex-direction: column;
+	`;
+
+	let flashcards = [];
+	const FetchPromise = fetch(`/usercards?folderKey=${props.folderKey}`, { method: 'GET' });
+	FetchPromise.then((result) => result.json())
+		.then((data) => {
+			flashcards = data.map((flashcard) => (
+				<Flashcard
+					key={flashcard.cardKey}
+					image={flashcard.blobKey}
+					text={flashcard.rawText}
+					translation={flashcard.textTranslated}
+					labels={flashcard.labels}
+				/>
+			));
+		})
+		.catch((error) => {
+			LogDebugMessage('Can not fetch flashcards.');
+		});
```

Comment:
I think you might have to write this page as a React Component. You should then call fetch inside of component did mount lifecycle because it was creating some bugs inside of Setup. You can look at my folderScroll.js file that does something similar to this.

Reply:
https://reactjs.org/docs/components-and-props.html#function-and-class-components

Reply:
ok!